pom.xml

```diff
@@ -721,6 +723,21 @@
                 <artifactId>spring-cloud-sap-core</artifactId>
                 <version>${hana.cloud.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.springframework</groupId>
+                <artifactId>spring-context-support</artifactId>

```

Comment:
Why do we need this new dependency? We should track it appropriately and request the usage.

Reply:
it is needed for quartz spring integration